frontend/public/locales/fr/apply-adult-child.json

```diff
@@ -190,8 +190,8 @@
     "federal-benefits": {
       "title": "Prestations fédérales",
       "legend": "Bénéficiez-vous d'une assurance dentaire dans le cadre d'un programme social offert par un programme fédéral?",
-      "option-yes": "<strong>Oui</strong>, je bénéficie d'une assurance <strong>fédérale</strong>",
-      "option-no": "<strong>Non</strong>, je ne bénéficie pas d'une assurance <strong>fédérale</strong>",
+      "option-yes": "<strong>Oui</strong>, je bénéficie d'une assurance <strong>fédérale</strong> dentaires",
+      "option-no": "<strong>Non</strong>, je ne bénéficie pas d'une assurance <strong>fédérale</strong> dentaires",
```

Comment:
@yannismassyl said to move `dentaires` before `<strong> and it's `dentaire` with no `s`. Do the same in other places.